cmd/ddltest/column_test.go

```diff
@@ -55,7 +56,7 @@ func (s *TestDDLSuite) checkAddColumn(c *C, rowID int64, defaultVal interface{},
 				// When insert a row with 3 columns, the third column value will be the first column value.
 				newInsertCount++
 			} else {
-				log.Fatalf("[checkAddColumn fail]invalid row: %v", data)
+				log.Fatal("[checkAddColumn fail]invalid row", zap.Any(`row`, data))
```

Comment:
I find it very strange that *every* string is introduced with `` `backticks` `` even if it does not contain `\` or `"`. The rest of the code uses plain `"double quoted"` strings so let's be consistent 

```suggestion
				log.Fatal("[checkAddColumn fail]invalid row", zap.Any("row", data))
```